### PR TITLE
Add simple jogging and update default view to include grid

### PR DESCRIPTION
**Pull request by pentacular**

### CODE REVIEW

File: api/shape/view.js [hunk 1]
```diff
@@ -58,7 +58,7 @@ export const baseView =
       height = 256,
       position = [100, -100, 100],
       withAxes = false,
-      withGrid = false,
+      withGrid = true,
```
Comment (BarbourSmith) [inline api/shape/view.js:61]: :)

File: ui/threejs/moveToFit.js [hunk 2]
```diff
@@ -16,7 +16,7 @@ export const moveToFit = ({
   scene,
   fitOffset = 1.2,
   withGrid = false,
-  gridLayer = SKETCH_LAYER,
+  gridLayer = GEOMETRY_LAYER,
```
Comment (BarbourSmith) [inline ui/threejs/moveToFit.js:19]: What does it mean that the grid is on the geometry layer now?

**Review by BarbourSmith — APPROVED** [review]
Looks great to me, I'm excited to try them 👍

File: ui/threejs/moveToFit.js (hunk 2 above)
Comment (pentacular) [reply ui/threejs/moveToFit.js:19]: It means that it won't be drawn over the top of geometry.

**Pull request merged**